Makefile

```
@@ -0,0 +1,28 @@
+.PHONY: conda-update
+conda-update:
+	envsubst < environment.yaml > /tmp/environment.yaml
+	conda env update -n saturn-client --file /tmp/environment.yaml
```

```suggestion
conda-update:
	conda env update -n saturn-client --file environment.yaml
```

Same as https://github.com/saturncloud/dask-saturn/pull/46